capstone/src/main/java/com/google/univiz/CarnegieSizeDegree.java

```diff
@@ -0,0 +1,31 @@
+package com.google.univiz;
+
+/**
+ * CarnegieSizeDegree is a class that holds the enum for the range of values for a school's carnegie
+ * size degree. The index of each of these values is the carnegie size degree provided by
+ * ScorecardData.
+ */
+public class CarnegieSizeDegree {
```

Comment:
Actually nvm I think you no longer need this, since `enum CarnegieSizeDegree` is syntactic sugar for `class CarnegieSizeDegree extends Enum`.